dex/msgjson/types.go

```diff
@@ -942,6 +974,86 @@ func (n *PenaltyNote) Serialize() []byte {
 	return append(b, []byte(p.Details)...)
 }
 
+// Client should send bond info when their bond tx is fully-confirmed. Server
+// should start waiting for required confs when it receives the 'postbond'
+// request if the txn is found. Client is responsible for submitting 'postbond'
+// for their bond txns when they reach required confs, or when the client
+// restarts and finds bonds recorded in their DB that is flagged as neither
+// accepted nor expired (also maybe if not listed in the 'connect' response).
+
+// PostBond should include the unsigned bond transaction for validation prior to
+// broadcasting the signed transaction so the client hasn't needlessly locked
+// funds if the transaction is rejected.
+type PostBond struct {
+	Signature
+	AcctPubKey Bytes  `json:"acctPubKey"` // acctID = blake256(blake256(acctPubKey))
+	AssetID    uint32 `json:"assetID"`
+	BondVer    uint16 `json:"bondVersion"`
+	BondTx     Bytes  `json:"bondTx"`
+	BondScript Bytes  `json:"bondScript"`
+}
```

Comment:
My instinct says we should just send the transaction ID, but since this pulls double duty as pre-validation, that doesn't work.